sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLPlannerImpl.java

```diff
@@ -98,27 +98,42 @@ public ZetaSQLPlannerImpl(FrameworkConfig config) {
 
   @Override
   public SqlNode parse(String s) throws SqlParseException {
-    return null;
+    throw new UnsupportedOperationException(
+        String.format(
+            "%s.parse(String) is not implemented and should need be called",
```

Comment:
nit: The message on these is long and unclear. What do you mean by "should need be called"? Could this whole message be simplified to "Not implemented"?

Reply:
Ah "should _not_ need to be called". But yes, it can be simplified. My intent was to signal to the client that this is a class with a bunch of extra methods most of which are not needed for the functionality it delivers.